src/otx/algo/utils/mmconfig.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Utils used for MMConfigs."""
+
+import inspect
+from pathlib import Path
+
+from omegaconf import DictConfig, OmegaConf
+
+
+def read_mmconfig(model_name: str, subdir_name: str = ".") -> DictConfig:
+    """Read MMConfig.
+
+    It try to read MMConfig from the yaml file which exists in
+    `<Directory path of __file__ who calls this function>/mmconfigs/<subdir_name>/<model_name>.yaml`
+    """
+    frame = inspect.stack()[1]
+    module = inspect.getmodule(frame[0])
+
+    if module is None or (mod_fpath := module.__file__) is None:
+        msg = "Cannot get valid model from stack"
```

Comment:
the error message is a bit too generic. can you add a more descriptive message so the user can understand what could be wrong.

Reply:
Fixed in https://github.com/openvinotoolkit/training_extensions/pull/2776/commits/669d935209ac14f857b52b25ce3957fc9ce03f15